po/it.po

```diff
@@ -0,0 +1,405 @@
+msgstr "Vai al fumetto precedente"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:29 cmd/xkcd-gtk/window.go:156
+msgid "Go to the next comic"
+msgstr "Vai al prossimo fumetto"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:36 cmd/xkcd-gtk/window.go:165
+msgid "Go to the newest comic"
+msgstr "Vai al fumetto più recente"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:43 cmd/xkcd-gtk/window.go:357
+msgid "Go to a random comic"
+msgstr "Vai ad un fumetto a caso"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:52 cmd/xkcd-gtk/window.go:299
+msgid "Search"
+msgstr "Cerca"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:56
+msgid "Search comics"
+msgstr "Cerca fumetti"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:65 cmd/xkcd-gtk/window.go:218
+msgid "Bookmarks"
+msgstr "Segnalibri"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:69
+msgid "Show bookmarks menu"
+msgstr "Mostra menu segnalibri"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:82
+msgid "Show comic properties"
+msgstr "Mostra proprietà fumetto"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:91
+msgid "Application Actions"
+msgstr "Azioni Applicazione"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:95
+msgid "Open new window"
+msgstr "Apri nuova finestra"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:102
+msgid "Toggle dark mode"
+msgstr "Abilità modalità scura"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:109
+msgid "Open shortcuts window"
+msgstr "Apri la finestra delle scorciatoie"
+
+#: cmd/xkcd-gtk/shortcuts-window.ui:116
+msgid "Quit application"
+msgstr "Esci dall'applicazione"
+
+#: cmd/xkcd-gtk/window.go:177
+msgid "Menu"
+msgstr "Menu"
+
+#: cmd/xkcd-gtk/window.go:182
+msgid "Open Link"
+msgstr "Apri Link"
+
+#: cmd/xkcd-gtk/window.go:183
+msgid "Explain"
+msgstr "Spiega"
+
+#: cmd/xkcd-gtk/window.go:237
+msgid "Bookmark this comic"
+msgstr "Aggiungi questo fumetto ai preferiti"
+
+#: cmd/xkcd-gtk/window.go:250
+msgid "Remove this comic from bookmarks"
+msgstr "Rimuovi questo fumetto dai preferiti"
+
+#: cmd/xkcd-gtk/window.go:325
+msgid "No results found"
+msgstr "Nessun risultato trovato"
+
+#: cmd/xkcd-gtk/window.go:353
+msgid "Random"
+msgstr "Casuale"
+
+#: cmd/xkcd-gtk/window.go:425
+msgid "Checking for new comic..."
+msgstr "Verifica di nuovi fumetti..."
+
+#: cmd/xkcd-gtk/window.go:458
+msgid "Loading comic..."
+msgstr "Caricamento fumetto..."
+
+#: cmd/xkcd-gtk/window.go:485
+msgid "Connect to the internet to download comic image"
+msgstr "Collegati a Internet per scaricare l'immagine del fumetto"
+
+#: data/com.github.rkoesters.xkcd-gtk.appdata.xml.in:10
+#: data/com.github.rkoesters.xkcd-gtk.desktop.in:7
+msgid "Read xkcd: a webcomic of romance, sarcasm, math, and language"
+msgstr "Leggi xkcd: un webcomic di romanticismo, sarcasmo, matematica e lingua"
+
+#: data/com.github.rkoesters.xkcd-gtk.appdata.xml.in:12
+msgid ""
+"Comic Sticks brings xkcd comics to your desktop with quick comic searching, "
+"offline browsing, and intuitive keyboard shortcuts."
+msgstr ""
+"Comic Sticks porta i fumetti xkcd sul desktop con una rapida ricerca dei fumetti",
```

Comment:
```suggestion
"Comic Sticks porta i fumetti xkcd sul desktop con una rapida ricerca dei fumetti, "
```

Having this comma outside the quotes is an error: https://travis-ci.org/rkoesters/xkcd-gtk/jobs/645175039#L706

Putting it inside the quotes should fix it (I'm assuming its grammatically correct Italian, but you would know better there)